### PR TITLE
Add a navbar

### PR DESCRIPTION
**Pull request by terrifricker**
**Closes Issue#14 - Add a navbar**

Add a simple nav bar with just a title "Chingu Voyage 24" and the name of our team "Team Toucan 03"

Using the google color picker, I chose a pretty yellow for the background of the nav, then chose a light and dark color from the same hue as a background color for main and for the border colors.  [My pretty yellow](https://g.co/kgs/Vwh4ZX)

Add the yellows as css variables in :root

### CODE REVIEW

**Review by Medic1111 — DISMISSED** [review]
Awesome job opening PR to dev!

**Review by alekzandriia — CHANGES_REQUESTED** [review]
I like it, but maybe instead of using absolute sizing units (px,pt), we could consistently use relative units (em, rem,etc.) so that it's responsive & looks pretty on every device? :)

**Comment by Medic1111** [comment]
> I like it, but maybe instead of using absolute sizing units (px,pt), we could consistently use relative units (em, rem,etc.) so that it's responsive & looks pretty on every device? :)

WOW, great job TEAM!!!!

File: index.html [hunk 1]
```diff
@@ -9,7 +9,10 @@
     </head>
     <body>
         <main>
-            <nav></nav>
```
Comment (terrifricker) [inline index.html:12]: I don't actually know how to accept this change right here in the GitHub review window.

Comment (terrifricker) [reply index.html:12]: Oh, this is actually the change that **I** made for the pull request.

**Comment by terrifricker** [comment]
Changes requested made.

Changed px units to responsive rem units for padding and gap.

**Review by alekzandriia — APPROVED** [review]

**Comment by terrifricker** [comment]
@alekzandriia  - I am now realizing that I didn't change the pt units.  How do we feel about leaving points as units for borders?

**Comment by alekzandriia** [comment]
> @alekzandriia  - I am now realizing that I didn't change the pt units.  How do we feel about leaving points as units for borders?

I noticed that. I think it's fine for small borders. In the official project, prior to pushing to main, we'd probably wanna test it out on multiple devices to be 💯 sure, but I think it's good. 😁

**Review by Medic1111 — APPROVED** [review]

**Review by QuangLyHo — APPROVED** [review]

**Pull request merged**